src/Integration/Binding/ISolutionBindingOperation.cs

```diff
@@ -36,7 +34,7 @@ namespace SonarLint.VisualStudio.Integration.Binding
     /// </remarks>
     public interface ISolutionBindingOperation : ISolutionBindingConfigFileStore
     {
-        void Initialize(IEnumerable<Project> projects, IDictionary<Language, SonarQubeQualityProfile> profilesMap);
```

Comment:
No longer needed, as it was only for the calculation of the BindingConfiguration.